IABTechLab/uid2-operator
# Ian UI d2 1713 load service service link

Testing local e2e:

PR to update uid2-dev-workspace with the needed localstack files.

https://gitlab.adsrvr.org/thetradedesk/teams/uid2/uid2-dev-workspace/-/merge_requests/10?resolved_conflicts=true

1. Within uid2-dev-workspace, checkout the following branches:

- admin: ian-UID2-1737-service-service-links-ui
- operator: ian-UID2-1713-load-service-service-link
- shared: ian-UID2-1677-service-service-links
- core: ian-UID2-1712-refresh-services-service_links

2. Update the pom.xml of shared/operator/admin to use the shared branch. Leave the core version the same  
3. Run docker compose all
4. Use the admin UI to make services and service links
5. Use operator temporary test endpoints /services and /service_links to see what operator has loaded
